instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_alveusOfTheHippocampus.jsonld

```diff
@@ -24,13 +24,18 @@
     },
     "hasParent": [
         {
-            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_hippocampalWhiteMatter"
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/hippocampal white matter"
```

Comment:
This needs to be changed back for all instances. I assume your script did something weird here 😉

Reply:
So I do not have this in my local copy, so I will commit it again and hopefully this will fix it then